common/config/rush/command-line.json

```diff
@@ -83,6 +83,14 @@
       "shellCommand": "(rush switch-flavor:stable && rush rebuild -t @azure/communication-react) || rush switch-flavor:beta && rush rebuild -t @azure/communication-react",
       "safeForSimultaneousRushProcesses": true
     },
+    {
+      "commandKind": "global",
+      "name": "generate-api-diff",
+      "summary": "Generate two api files for a supplied feature name",
+      "description": "Generate two api files that differ only by a supplied feature name. Note: This result of this commands will switch current flavor to beta.",
```

Comment:
It would be great to add a description of the generated files (as in `common/scripts/generate-api-diff.mjs`) so if someone doesn't use the command often, they still will understand the purpose for both files without searching in code.

Reply:
Great point! Updated the description with more details